CONTRIBUTING.md

```diff
@@ -206,6 +206,11 @@ For reference implementation, check [push-sbom-to-pyxis/tests/](/tasks/push-sbom
       check mock calls after task execution in our test pipeline.
     - In this case the function touches a file that would otherwise be created by the actual `cosign`
       call. This is specific to the task and will depend on your use case.
+    - Note: In the example above, the function being mocked is `cosign`. If that function was actually something
```

Comment:
Can you add a bit about why it fails? e.g. "Note: In the example above, the function being mocked is `cosign`. If that function was actually something that had a hyphen in its name (e.g. `my-cool-function`), the tests would fail with `my-cool-function: not a valid identifier` messages. This is because when you use `#!/usr/bin/env sh`, Bash apparently runs in POSIX mode in which case hyphens are not permitted in function names. The solution is to use `#!/bin/bash` or `#!/usr/bin/env bash` and then it will work. Also, keep in mind that the same shebang should be used in both the mock and the actual step script that you're mocking to ensure the behavior during test is the same as during runtime."

Reply:
Updated